hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegionServer.java

```diff
@@ -1460,6 +1466,10 @@ RegionLoad createRegionLoad(final HRegion r, RegionLoad.Builder regionLoadBldr,
       totalStaticIndexSizeKB += (int) (store.getTotalStaticIndexSize() / 1024);
       totalStaticBloomSizeKB += (int) (store.getTotalStaticBloomSize() / 1024);
     }
+    //HBASE-26340 Fix false "0" size under 1MB
+    if(storefileSizeMB < 1 && nonEmptyStoreExist) {
```

Comment:
Do we need to apply this trick to other fields as well? Just introduce a round method

```
long round(long size, long unit) {
  // normally return size / unit, if size is smaller than unit but greater than 0, return 1
}
```

And all the size should be calculated by this method

Reply:
I'm not aware we need to do this to other fields as well. I agree we can leave "nonEmptyStoreExist" out, but I would also think a simple ? : does the job here, no need for a method. I'll update the PR.

Reply:
At region server side we should know the accurate size in bytes, so when calculating we could just cumulate the size in bytes, and at last we convert them to size in KB or size in MB or what ever unit. WDYT?

Reply:
That works, sure, at least I think we are thinking the same when I said above we could use long to track "storefileSize(MB)(Temp)" - no need for Temp though just remove MB - but at the end we set this as MB just like before. I'll do the change today.